### PR TITLE
Fixes #19 Decouple models and utils completely, implement MVC

### LINKED ISSUE #19: decouple models and utils completely

### PR DESCRIPTION
**Pull request by zeyus**
Fixes #19, now implemented MVC style

### CODE REVIEW

**Comment by zeyus** [comment]
🌟⭐🎈🎆🎇🧨✨🎉🎊🎃🎃🎃🎃

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
         pip install -r requirements-dev.txt
     - name: Analysing the code with pylint
       run: |
-        pylint --load-plugins pylint_flask_sqlalchemy $(git ls-files '*.py')
```
Comment (zeyus) [inline .github/workflows/push.yml:23]: now gets config from `pyproject.toml`

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -3,5 +3,5 @@ repos:
     rev: pylint-2.6.0
     hooks:
     -   id: pylint
-        args: [--load-plugins, pylint_flask_sqlalchemy]
-        additional_dependencies: [pylint-flask, pylint-flask-sqlalchemy, pytest]
+        args: [--load-plugins, pylint_flask, -j4]
```
Comment (zeyus) [inline .pre-commit-config.yaml:6]: seems we just need `pylint_flask` and all is good!

**Review by emilroenn — APPROVED** [review]
Looks good, is very clean, and nothing breaks - I'm glad we had an in-person rundown

**Pull request merged**